### DISCUSSION TITLE
Deploy YOLOv8 with TensorRT | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#968] [Q&A]
# Deploy YOLOv8 with TensorRT | Seeed Studio Wiki

Deploy YOLOv8 on NVIDIA Jetson using TensorRT - Data Label, AI Model Train, AI Model Deploy

https://wiki.seeedstudio.com/YOLOv8-TRT-Jetson/

**Comment by isabellagreen968** [comment]
Very useful deployment guide! Using TensorRT to run YOLOv8 on NVIDIA Jetson significantly improves inference speed and efficiency, making it ideal for edge AI solutions. Pairing [YOLOv8 and DeepStream](https://yolov8.org/yolov8-and-deepstream/) further enhances performance, enabling real-time video analytics with minimal latency. This combo is definitely a powerful choice for production-ready object detection systems.
